client/scripts/sw-docker-entrypoint

```diff
@@ -108,6 +108,16 @@ serve() {
     fi
 }
 
+fine_tune() {
+    echo "-->[FineTuning] start to fine tune, use $(which swcli) cli ..."
```

Comment:
"fine-tune" is the correct word to use when referring to the process of optimizing a pretrained model, so we should use `Fine-Tuning` to replace `FineTuning`?